medicines/search-client/docs/development-and-releasing.md

```diff
@@ -0,0 +1,15 @@
+# Developing and Releasing this Package
+
+## Release Candidates
+
+To publish a new, release candidate version of this package:
+
+1. Set a new version number in `Cargo.toml` with an `-rc` suffix (for example, `1.2.2` is incremented to `1.2.3-rc1`, `1.2.3-rc4` is incremented to `1.2.3-rc5`).
```

Comment:
Is this file still needed if we're not pushing to crates.io anymore?

Reply:
Nope, good point.